src/common/libsubprocess/test/test_multi_echo.c

```diff
@@ -36,19 +45,57 @@ main (int argc, char *argv[])
         }
     }
 
+    if ((out + err) == 0) {
+        fprintf (stderr, "must specify a way to output");
+        exit (1);
+    }
+
     if (optind != argc) {
```

Comment:
I think the while loop takes care of what this conditional is checking for (remove?)